docs/slack.rst

```diff
@@ -259,22 +259,48 @@ In addition to the guidance that applies to all participants, we expect self-pro
   If you're running a survey, say when and how the results will be published.
+
 - **Give more than you take.**
   Make self-promotion a reasonable proportion of your participation in the community.
   Too much self-promotion creates distrust and hostility.
   If you're not sure if it's too much, aim to help others 10 times for every time you self-promote.
+
 - **Post publicly instead of sending direct messages (DMs)**.
   A DM is like a home and you're like a vampire: you're not allowed in without an invitation.
   It can be tempting to DM people, but please respect their inboxes and post in public channels unless invited to DM.
+
 - **Read the room.**
   The Write the Docs Slack is informal and conversational; high-pressure sales aren't welcome.
   If people aren't interested in what you're offering, don't take it as a license to become louder and more aggressive.
   If you're asked to stop, then stop.
+
 - **Treat the community like friends, not customers.**
   Posts that resemble infomercials are harder for the community to engage with and for the moderation team to distinguish from spam.
   If you post about things that aren't relevant to the documentation field or the channel you're posting in, moderators might delete your messages.
   Make sure conversations stay relevant and not sales-focused, so you can have the chance to talk about work you're excited about.
 
+Here are some examples of common self-promotion situations, and some guidelines on how to self-promote responsibly:
+
+You've written a blog post (Medium, Hashnode, etc.) that you want to share with the community, but it's not closely related to an ongoing discussion
+  **DO** share it in #community-showcase and tell us why you think it would be interesting or relevant to documentarians
+  
+  **DON'T** cross-post that content across multiple channels
+
+Your employer is hosting an event to promote their product or service (even if it's not the primary purpose of the event)
+  **DO** share it in #community-events and tell others your connection to the event
+  
+  **DON'T** use "marketing language" to obscure the purpose of the event, your relationship to the employer, or the costs associated to attendance
+
+Someone else has asked a question and you're the author of an article, blog, book, or similar resource that answers the question
+  **DO** reply by briefly summarizing your article, noting your authorship, and linking to the article
+  
+  **DON'T** solicit views/clickthroughs/purchases of your content, especially via Direct Messages or cross-posting out-of-context
+
+You are offering cash for gold, selling diet supplements, representing the estate of a recently diseased and heretofore unknown family member, or doing some other activity unrelated to documentation but for some reason requires our members to DM you their bank account information
+  **DO** consider better ways to engage with this community
+  
+  **DON'T** post that content to Write the Docs
+
```

Comment:
OK, I realize this joke was my suggestion but seeing it written out in context, I think we might need to kill our darlings here. Nobody who has this intent is going to read the guidelines and we're just going to delete it anyway. I suggest that this doesn't actually serve a plausible audience. 😞

```suggestion

```

Reply:
:-D but it goes give you something to link to when you do delete it. :-D